### ISSUE TITLE
macOS: Mount at /Volumes/keybase instead of /

### ISSUE THREAD

**Issue by jsejcksn** [keybase/client#3349]
Please, it's the standard location.

**Comment by jsejcksn** [comment]
Also, Keybase cannot mount if the user already has a folder named `/keybase`. See log:

`c8415a0342b70365ddfb6a1c`

**Comment by cjb** [comment]
We'd like KBFS paths to be shareable (e.g. in scripts) between Linux and macOS, so using a place that is only standard on the Mac wouldn't work for us.

**Comment by jsejcksn** [comment]
But what about working for the community?

**Comment by MasterRoot24** [comment]
👍 from me to use the **standard** place to mount filesystems on Mac OS: `/Volumes`.

**Comment by MattiSG** [comment]
@cjb What about automatically adding a symlink `/Volumes/keybase → keybase`?  😃 
I was also quite surprised to have a mount point of `/`.

**Comment by cjb** [comment]
@MattiSG Ah, maybe try it and let us know how it works?

**Comment by MasterRoot24** [comment]
Great idea from @MattiSG about the symlink!

I'd like to expand that idea and suggest that to me it would make more sense to mount KBFS in the OS directory of choice (`/mnt/keybase` on Linux, `/Volumes/keybase` on Mac) and create a symlink of `/keybase` to provide the common cross-platform KBFS location?

**Comment by MattiSG** [comment]
I agree that compliance should come first, and convenience second. Both are crucial to adoption, but let's make that symlink a nicety to users and hard links an unambiguous point for the system (should especially come in handy for system upgraders).
I'd be in favor of a system-specific prefix for the `keybase` mount point, and a `/keybase` symlink pointing to the proper mount point for portability.
